src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -21,17 +21,87 @@ Public Class ParseDirectives
+        ]]>, TestOptions.Script)
+        ParseAndVerify("# !/usr/bin/env vb", TestOptions.Script,
+                       <errors><error id="30248" start="1" end="1"/></errors>)
+        ParseAndVerify("#/usr/bin/env vb", TestOptions.Script,
+                       <errors><error id="30248" start="1" end="1"/></errors>)
+    End Sub
+
+    Public Sub ParseShebangInRegularCode()
+        ParseAndVerify("#!", TestOptions.Regular,
+                       <errors><error id="37254" message="#! directive is only allowed is script code." start="0" end="1"/></errors>)
+    End Sub
+
+    Public Sub ParseShebangDirectiveHasToBeAtTopOfTheFile()
+        ParseAndVerify(" #!/usr/bin/env vb", TestOptions.Script,
+                       <errors><error id="37255" message="#! directive should appear as the first directive in the script file without any whitespace." start="1" end="1"/></errors>)
+        ParseAndVerify("
+#!/use/bin/env vb", TestOptions.Script,
+                       <errors><error id="37255" message="#! directive should appear as the first directive in the script file without any whitespace." start="1" end="1"/></errors>)
+        ParseAndVerify(
+"Dim x = 12
+#!/use/bin/env vb", TestOptions.Script,
+                       <errors><error id="37255" message="#! directive should appear as the first directive in the script file without any whitespace." start="1" end="1"/></errors>)
+    End Sub
+
     <Fact>
     Public Sub ParseReferenceDirective()
         ParseAndVerify(<![CDATA[
-            #r "reference"
+            #r "assembly.dll"
+        ]]>, TestOptions.Script)
+
+        ParseAndVerify(<![CDATA[
+            #r "assembly.dll" 'comment
+        ]]>, TestOptions.Script)
+
+        ParseAndVerify(<![CDATA[
+            #r "assembly.dll" Dim x = 1
+        ]]>, TestOptions.Script,
+        <errors>
+            <error id="30205" message="End of statement expected." start="26" end="29"/>
+        </errors>)
+
+        ParseAndVerify(<![CDATA[
+            #r "assembly.dll"
+        ]]>,
+        <errors>
+            <error id="36964" message="#R Is only allowed in scripts" start="14" end="15"/>
+        </errors>)
+    End Sub
+
+    <Fact>
+    Public Sub ParseLoadDirective()
+        ParseAndVerify(<![CDATA[
+            #load "script.vbx"
+        ]]>, TestOptions.Script)
+
+        ParseAndVerify(<![CDATA[
+            #load "script.vbx" 'comment
         ]]>, TestOptions.Script)
 
         ParseAndVerify(<![CDATA[
-            #r "reference"
+            #load "script.vbx" Dim x = 1
+        ]]>, TestOptions.Script,
+        <errors>
+            <error id="30205" message="End of statement expected." start="26" end="29"/>
+        </errors>)
+
+        ParseAndVerify(<![CDATA[
+            #load "script.vbx"
         ]]>,
         <errors>
-            <error id="36964" message="#R is only allowed in scripts" start="14" end="15"/>
+            <error id="36967" message="#Load Is only allowed in scripts" start="14" end="18"/>
```

Comment:
It looks like this test helper somewhat deceivingly fails to check the message (this test really should fail).  Perhaps it should at least be modified to check that the message is not empty string.

In any case, you need to add this resource string (and note the typo:  "Is" vs "is").